### ISSUE TITLE
Connection popup doesn't fit the screen anymore

### ISSUE THREAD

**Issue by vortex36** [ThePBone/GalaxyBudsClient#60]
After updating the client to 3.1.1 from 3.0, the connection popup isn't displayed correctly anymore, and acts as if the screen is bigger than it is. What I mean by that is that any position besides top-left is moved more to the right (and in the case of bottom positions, more to the bottom).

This is the top center position:
![immagine](https://user-images.githubusercontent.com/46104112/99642028-514f5700-2a4b-11eb-967d-2790a1886865.png)

This is bottom left:
![immagine](https://user-images.githubusercontent.com/46104112/99642113-688e4480-2a4b-11eb-9f16-804f4974f831.png)

I have the screen items/text resized at 125% through Windows 10's settings, so I thought that that could cause this issue, but even if I set it at 100% the issue was still there.

**Comment by ThePBone** [comment]
I did some tests and can confirm that the scaling setting causes this issue. I recently moved to another "screen API" that changed how the popup retrieves its on-screen coordinates (In order to show the popup on all screens on a multi-monitor setup). Apparently, this API ignores any scaling preferences and works very inconsistently if the PC uses a non-100% scaling setting.

I'll probably move back to the old implementation since there aren't any other good .net APIs I could use for this (except using unsafe p/invoke calls to unmanaged C++ APIs).

I'll post an update as soon as I've reverted these changes.
